cmd/cloudshell_open/main.go

```diff
@@ -291,14 +292,24 @@ func run(opts runOpts) error {
 		}
 	}
 
+	end = logProgress(
+		fmt.Sprintf("Setting up %s in region %s (if it doesn't already exist)", highlight(artifactRegistry), highlight(region)),
+		fmt.Sprintf("Set up %s in region %s (if it doesn't already exist)", highlight(artifactRegistry), highlight(region)),
+		"Failed to setup artifact registry.")
+	err = createArtifactRegistry(project, region, artifactRegistry)
+	end(err == nil)
+	if err != nil {
+		return err
+	}
+
 	repoName := filepath.Base(appDir)
 	serviceName := repoName
 	if appFile.Name != "" {
 		serviceName = appFile.Name
 	}
 	serviceName = tryFixServiceName(serviceName)
```

Comment:
Based on previous suggestion about the function returning an error, then this should check for err != nil...